src/test/java/com/wikia/webdriver/testcases/contentreview/ContentReviewTests.java

```diff
@@ -1,5 +1,6 @@
 package com.wikia.webdriver.testcases.contentreview;
 
+import org.testng.annotations.Test;
 import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)